### PR TITLE
Fix internal links (plus other housekeeping)

### PR DESCRIPTION
**Pull request by willfurnass**
Partly addresses #237 (need separate PR to fix external links where possible).

### CODE REVIEW

**Comment by willfurnass** [comment]
Broken *internal* links that still need to be fixed (these were identified using https://github.com/stevenvachon/broken-link-checker):

```
$ node_modules/.bin/blc http://localhost:8080 --recursive  --exclude-external | grep -v ─OK─ | grep -B 2 BROKEN

Getting links from: http://localhost:8080/service/
├─BROKEN─ http://localhost:8080/support (HTTP_404)
--

Getting links from: http://localhost:8080/training/deeplearning
├─BROKEN─ http://localhost:8080/training/deeplearning/deeplearning.net/software/theano (HTTP_404)
--

Getting links from: http://localhost:8080/community
├─BROKEN─ http://localhost:8080/blog/SSI-2018 (HTTP_404)
--

Getting links from: http://localhost:8080/community/resources-and-equipment
├─BROKEN─ http://localhost:8080/community/resources-and-equipment/languages/NAG (HTTP_404)
├─BROKEN─ http://localhost:8080/community/resources-and-equipment/languages/rstats (HTTP_404)
--

Getting links from: http://localhost:8080/blog/2020-03-24-configuring-cuda-and-opengl-courses-in-the-cloud-md/
├─BROKEN─ http://localhost:8080/blog/2020-03-24-configuring-cuda-and-opengl-courses-in-the-cloud-md/www.instancehub.com (HTTP_404)
├─BROKEN─ http://localhost:8080/blog/2020-03-24-configuring-cuda-and-opengl-courses-in-the-cloud-md/www.instancehub.com (HTTP_404)
--

Getting links from: http://localhost:8080/blog/learn-python/
├─BROKEN─ http://localhost:8080/blog/learn-python/rosalind.info (HTTP_404)
--

Getting links from: http://localhost:8080/blog/xarray-for-earth/
├─BROKEN─ http://localhost:8080/blog/xarray-for-earth/www.github.com/jmcook1186/IceSurfClassifiers/ (HTTP_404)
--

Getting links from: http://localhost:8080/blog/hacktoberfest-2018/
├─BROKEN─ http://localhost:8080/blog/sheffieldR-hacktoberfest/ (HTTP_404)
--

Getting links from: http://localhost:8080/blog/ssi-2018/
├─BROKEN─ http://localhost:8080/blog/linuistics_grant_2016/ (HTTP_404)
├─BROKEN─ http://localhost:8080/blog/intel-R-iceberg/ (HTTP_404)
--

Getting links from: http://localhost:8080/blog/soft-carp-at-tuos/
├─BROKEN─ http://localhost:8080/2017-08-16-sheffield/ (HTTP_404)
--

Getting links from: http://localhost:8080/community/
├─BROKEN─ http://localhost:8080/blog/SSI-2018 (HTTP_404)
--

Getting links from: http://localhost:8080/community/resources-and-equipment/languages/maple
├─BROKEN─ http://localhost:8080/blog/HPC-Maple-1/ (HTTP_404)
```

File: _posts/2016-07-02-the-university-of-sheffield-named-an-nvidia-gpu-education-center.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ type: text
 
 This week I am very pleased to announce that the University of Sheffield has been awarded the status of an NVIDIA CUDA Education Centre.
 
-The faculty of Engineering has featured this in its latest [faculty newsletter](https://www.sheffield.ac.uk/faculty/engineering/news/nvidia-1.587003) and the Department of Computer Science has published more details in a [news feature](https://www.sheffield.ac.uk/dcs/latest-news/nvidia-1.587214).
```
Comment (Robadob) [inline _posts/2016-07-02-the-university-of-sheffield-named-an-nvidia-gpu-education-center.md:17]: Whilst google lists https://www.sheffield.ac.uk/faculty/engineering/news/nvidia-1.587003, it redirects to engineering home page. Might want to remove this link too

File: _posts/2017-03-20-sge-job-validation.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ when things go wrong.
 For example, on our ShARC cluster, which runs the (Son of) Grid Engine (SGE) scheduler, 
 if you request a longer run-time than is permitted by any of the cluster's job queue configurations 
 then your job will sit there queueing indefinitely until you or someone else deletes it.  
-For example, let's use [qsub](http://docs.hpc.shef.ac.uk/en/latest/hpc/scheduler/sge.html#running-batch-jobs-on-iceberg)
```
Comment (Robadob) [inline _posts/2017-03-20-sge-job-validation.md:23]: This link is 404, do you know a suitable replacement

File: _posts/2017-06-15-tmux-intro.md [hunk 3]
```diff
@@ -300,17 +300,17 @@ Using tmux on HPC clusters
...
-Iceberg](http://docs.hpc.shef.ac.uk/) (assuming that you don't need a
+Iceberg](https://docs.hpc.shef.ac.uk/) (assuming that you don't need a
 [GUI](https://en.wikipedia.org/wiki/Graphical_user_interface)).
 
 On ShARC and Iceberg can:
 
 1.  Start a tmux or GNU Screen session on a login node;
 2.  Start an interactive job using [qrshx or
-    qrsh](http://docs.hpc.shef.ac.uk/en/latest/hpc/scheduler/sge.html);
+    qrsh](https://docs.hpc.shef.ac.uk/en/latest/hpc/scheduler/sge.html);
```
Comment (Robadob) [inline _posts/2017-06-15-tmux-intro.md]: 404

File: _posts/2017-06-15-tmux-intro.md [hunk 4]
```diff
@@ -300,17 +300,17 @@ Using tmux on HPC clusters
 ==========================
 
 Terminal Multiplexers can be useful if doing [interactive
-work](http://docs.hpc.shef.ac.uk/en/latest/hpc/scheduler/sge.html#interactive-sessions))
+work](https://docs.hpc.shef.ac.uk/en/latest/hpc/scheduler/sge.html#interactive-sessions))
```
Comment (Robadob) [inline _posts/2017-06-15-tmux-intro.md]: Same 404 as other comments

File: _posts/2020-03-24-configuring-cuda-and-opengl-courses-in-the-cloud.md [hunk 5]
```diff
@@ -13,15 +13,15 @@ type: text
...
 
-The final option is to install the CUDA driver (which will run in TCC mode) and then switch the driver mode to the windows driver display model (WDDM) rather than TCC. This mode is slightly less performant than TCC mode but is used for consumer and Quadro cards which drive displays as well as perform CDUA compute. Through trial and error this will only work on certain Tesla cards. K80s are not supported but M60s do. By running the following command (from the C:\Program Files\NVIDIA Corporation\NVSMI location as Administrator) you can put the M60 device into WDDM mode after a reboot.
+The final option is to install the CUDA driver (which will run in TCC mode) and then switch the driver mode to the windows driver display model (WDDM) rather than TCC. This mode is slightly less performant than TCC mode but is used for consumer and Quadro cards which drive displays as well as perform CUDA compute. Through trial and error this will only work on certain Tesla cards. K80s are not supported but M60s do. By running the following command (from the C:\Program Files\NVIDIA Corporation\NVSMI location as Administrator) you can put the M60 device into WDDM mode after a reboot.
```
Comment (Robadob) [inline _posts/2020-03-24-configuring-cuda-and-opengl-courses-in-the-cloud.md]: Kind of unrelated, but you might want to wrap the system path in `markdown code`

**Comment by Robadob** [comment]
Made a few notes, couldn't spot any other issues. (I didn't check internal link changes)

**Pull request merged**